MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -45,6 +45,20 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
      */
     @objc public var abbreviationPriority: Int = NSNotFound
     
+    /**
+     An array indicating which directions you can go from a lane (left, right, or straight).
+     
+     If the value is `[.left", .straight]`, the driver can go straight or left from that lane. This is only set when the `component` is a lane.
+     */
+    @objc public var directions: [Any]?
```

Comment:
`Any` was used instead of `ManeuverDirection` -- since the `objc` declared property is an array of enum.

Reply:
How about renaming this and making it a lane indication: https://github.com/mapbox/MapboxDirections.swift/blob/master/MapboxDirections/MBLaneIndication.h